src/core/util/prefixer.js

```diff
@@ -63,10 +67,30 @@ function MdPrefixer(initialAttributes, buildSelector) {
   }
 
   function _removeAttribute(element, attribute) {
-    element = element[0] || element;
+    element = _parseElement(element);
+
+    if (!element) {
+      return;
+    }
 
     _buildList(attribute).forEach(function(prefixedAttribute) {
       element.removeAttribute(prefixedAttribute);
     });
   }
+
+  /**
+   * Parses a jqLite or DOM element into a HTML element.
+   * This is useful when supporting jqLite elements and DOM elements at
+   * same time.
+   * @param element {JQLite|Element} Element to be parsed
+   * @returns {HTMLElement} Parsed HTMLElement
+   */
+  function _parseElement(element) {
```

Comment:
`parse` isn't really the right word. It implies that you're taking a string recognizing it into something else. I'd use `_getNativeElement`